File: prime_diffarance.py
arr = [2, 3, 5, 7, 9]  # example array
result = [0]  # to store the differences

for i in range(len(arr)-1):
    diff = arr[i+1] - arr[i]  # calculate the difference
    result.append(diff)  # add the difference to the result array

print(result)  # output: [2, 2, 2, 2]

File: prime_diffarance.js
const arr = [2, 4, 6, 8, 10]; // example array
const result = []; // to store the differences

for (let i = 0; i < arr.length - 1; i++) {
  const diff = arr[i+1] - arr[i]; // calculate the difference
  result.push(diff); // add the difference to the result array
}

console.log(result); // output: [2, 2, 2, 2]
